language-agent/JVMMetric.proto

```diff
@@ -83,6 +83,7 @@ message GC {
 enum GCPhrase {
     NEW = 0;
     OLD = 1;
+    NORMAL = 2;
```

Comment:
Please add comments about what is a normal GC phase.

Reply:
where should i comment，near this code 

like 
 NEW = 0;
    OLD = 1;
    NORMAL = 2;  // comments

Reply:
It is fine either like your style or this style.

```suggestion
    // ....
    NORMAL = 2;
```

Reply:
ok